app/uk/gov/hmrc/constructionindustryscheme/services/chris/ChrisSubmissionXmlMapper.scala

```diff
@@ -70,5 +56,28 @@ object ChrisSubmissionXmlMapper extends ChrisXmlMapper {
       SubmissionResult(status, xml, meta)
     }
   }
-  
+
+  /** Stage 1 (initial submit) status mapping – ACK or FATAL only. */
+  private def deriveInitialStatus(
+                                   qualifier: String,
+                                   errOpt: Option[GovTalkError]
+                                 ): SubmissionStatus =
+    qualifier.toLowerCase match {
+      case "acknowledgement" =>
+        ACCEPTED
+
+      case "error" =>
+        errOpt match {
+          case Some(err)
+            if err.errorNumber == "3000" &&
+              err.errorType.equalsIgnoreCase("fatal") =>
```

Comment:
unit test for condition:
if err.errorNumber == "3000" && err.errorType.equalsIgnoreCase("fatal")